### PR TITLE
Documentation clarification.

### PR DESCRIPTION
**Pull request by roconnor-blockstream**

### CODE REVIEW

**Review by sanket1729 — APPROVED** [review]
ACK efda3146dd7161f806817b063a2a62bf2cf6f4f6

File: C/include/simplicity/elements/env.h [hunk 1]
```diff
@@ -34,8 +34,8 @@ typedef struct rawOutput {
   rawBuffer rangeProof;
 } rawOutput;
 
-/* A structure representing data for one input from an Elements transaction, plus the TXO data of the output being redeemed,
- * plus all the segwit annexes.
+/* A structure representing data for one input from an Elements transaction, including it's taroot annex,
```
Comment (sanket1729) [inline C/include/simplicity/elements/env.h:37]: nit: /s/taroot/taproot

Comment (roconnor-blockstream) [reply C/include/simplicity/elements/env.h]: Fixed.

File: C/include/simplicity/elements/env.h [hunk 2]
```diff
@@ -34,8 +34,8 @@ typedef struct rawOutput {
   rawBuffer rangeProof;
 } rawOutput;
 
-/* A structure representing data for one input from an Elements transaction, plus the TXO data of the output being redeemed,
- * plus all the segwit annexes.
+/* A structure representing data for one input from an Elements transaction, including it's taproot annex,
```
Comment (apoelstra) [inline C/include/simplicity/elements/env.h:37]: `it's` should be `its`

**Review by apoelstra — APPROVED** [review]
utACK 198ceabc38856fc7e25cb87f0eb9082713ab01e2

**Pull request merged**